cypress/fixtures/secondarySingleMovieData.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
A second data set so that we could test for another movie (safety net) and so that we could make sure everything went through our cleaning function but nothing was altered since nothing needed alteration.